applications/convection_diffusion_application/tests/test_ConvectionDiffusionApplication.py

```diff
@@ -1,14 +1,34 @@
 # import Kratos
-from KratosMultiphysics import *
-from KratosMultiphysics.ConvectionDiffusionApplication import *
+import KratosMultiphysics
+import KratosMultiphysics.ConvectionDiffusionApplication
+import run_cpp_unit_tests
 
 # Import Kratos "wrapper" for unittests
 import KratosMultiphysics.KratosUnittest as KratosUnittest
 
+import subprocess
```

Comment:
Maybe a stupid question, but where are subprocesses used?

Reply:
Huuuum, copy/paste from structural

Reply:
So, can we remove it?

Reply:
I assume, @philbucher what do you think?

Reply:
yes pls remove

only needed for launching the mpi tests in the structural (which is for sure only a temp solution!)

Reply:
Ok